src/library_idbstore.js

```diff
@@ -125,6 +139,15 @@ var LibraryIDBStore = {
       });
     });
   },
+  emscripten_idb_clear__async: true,
+  emscripten_idb_clear: (db, perror) => {
+    return Asyncify.handleSleep(function(wakeUp) {
+      IDBStore.clearStore(UTF8ToString(db), function(error) {
+        {{{ makeSetValue('perror',  0, '!!error', 'i32') }}};
```

Comment:
Looks like two spaces before the `0`

Reply:
Can you use arrow functions here.  I fixed up the rest of this file in #20200

Reply:
Sorry for my late response, it should be done now (I hope you don't mind force pushes)